src/styles/lines/lines.js

```diff
@@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
 import Geo from '../../geo';
 import Utils from '../../utils/utils';
 
+let fs = require('fs');
+let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
```

Comment:
Leading towards exporting the shaders from the polygons module and importing them from lines.js

```
// polygons.js
export {
    shaderSrc_polygonsVertex,
    shaderSrc_polygonsFragment
};
```

```
// lines.js
import {shaderSrc_polygonsVertex, shaderSrc_polygonsFragment} from '../polygons/polygons';
```

Thoughts?